### ISSUE TITLE
Make the menu persistent when chancing between the menu options

### ISSUE THREAD

**Issue by g13johbj** [HGustavs/LenaSYS#2025]
It is annoying that you need to go back to the course homepage when you want to see the menu and / or switch between pages within that course. 
I think it would be a good idea to make this menu persist and always be visible in all the different sub pages of a course.

![menuissue](https://cloud.githubusercontent.com/assets/9322288/14317149/2b4268d0-fc07-11e5-924c-5c2fe961b83b.png)

**Comment by g13johbj** [comment]
@HGustavs @a97marbr , is this ok?

**Comment by a97marbr** [comment]
No I don't want that, since different sub-pages have different needs. I do however like the idea of moving the menu to the top-list as with the CodeViewer.

I would also like to have icons for the menu options, or perhaps a button with both icon **and** text if we have space, and only icons when space is an issue.

**Comment by HGustavs** [comment]
By moving those buttons into the top menu (but only for this page) we get more space that we can use do display the forum. 

What can the icons look like?

* For results we can reuse the fist icon
* Dont we have a files icon for the code viewer?
* Test and access needs new icons?

**Comment by a97marbr** [comment]
We have two files icons. None of which is good (read material design-ish)
* https://github.com/HGustavs/LenaSYS/blob/master/Shared/icons/files_iconOld.svg (I would like something along these lines but with a better material design look)
* https://github.com/HGustavs/LenaSYS/blob/master/Shared/icons/files_icon.svg

Yes I don't know that we have any test or access icons

**Comment by g13johbj** [comment]
Since the space pointed out in this picture is never used for anything else, should the menu be moved up there or on the other side like in the codeviewer?

On the side, pointed out here, both icons and text can fit without no problem. Or we can just have icons with explaining text when hovering over the icons.

![menuissue1](https://cloud.githubusercontent.com/assets/9322288/14331466/a8bc784e-fc45-11e5-91fc-ee79f043ac9f.png)

**Comment by a97marbr** [comment]
I kind of like the idea with both icon and text if we have space ... considering that a newer tablet have pretty good resolution (and no hover)...

**Comment by a97marbr** [comment]
Yeah, and I would follow the layout of CodeViewer ... consistency for the win

**Comment by g13johbj** [comment]
I'm guessing that we need to make our own icons and cannot use icons from googles material icons library for the web?

**Comment by g13johbj** [comment]
http://google.github.io/material-design-icons/#getting-icons

**Comment by HGustavs** [comment]
Well. If there is one that fits the existing ones stylistically and that happens to be CC (any use) or PD then it is ok.

**Comment by HGustavs** [comment]
There must be re-distribution-permitted CC, not only attribution as we must redistribute the work.

**Comment by HGustavs** [comment]
Own icons is always safer as it by definition is redistributable under the lenasys license.

**Comment by g13johbj** [comment]
"We'd love attribution in your app's about screen, but it's not required. The only thing we ask is that you not re-sell the icons themselves."

This is from the section under Licensing. From that I take that we don't need to give any credit or anything alike, just not sell the icons.

**Comment by g13johbj** [comment]
The idea is to make the menu persistent and move it up to the navbar so you don't need to go back when switching between files and tests for example.

**Comment by a15freed** [comment]
#1823
